.vue.webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const webpack = require('webpack');
+const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const VueLoaderPlugin = require('vue-loader').VueLoaderPlugin;
+
+const config = {
+    entry: {
+        'vue': './lib/bundles/vue.js',
+    },
+    output: {
+        path: path.resolve(__dirname, 'public/lib/vue'),
+        publicPath: '/public/lib/vue/',
+        asyncChunks: true,
+        clean: true,
+    },
+    module: {
+        rules: [
+            {
+                // Vue SFC
+                test: /\.vue$/,
+                loader: 'vue-loader'
+            },
+            {
+                // Build styles
+                test: /\.css$/,
+                use: [MiniCssExtractPlugin.loader, 'css-loader'],
```

Comment:
Not tested, but I guess that without this plugin, the CSS code will be embed in the built component JS file and added at runtime when the component will be loaded/rendered. As the CSS code prevent in vue SFC is often minimalist, it will not be a problem to handle it with javascript code, and it will permit to preserve HTTP queries when loading components.

```suggestion
                use: ['css-loader'],
```

Reply:
:+1: